src/java.base/unix/classes/sun/security/provider/NativePRNG.java

```diff
@@ -385,11 +466,12 @@ private static class RandomIO {
         private final Object LOCK_SET_SEED = new Object();
 
         // constructor, called only once from initIO()
-        private RandomIO(File seedFile, File nextFile) throws IOException {
+        private RandomIO(File seedFile, File nextFile, final Variant flag) throws IOException {
```

Comment:
It's more important that the field is final than the parameter, which should be named `variant` to match.

Reply:
updated